index.qmd

```diff
@@ -9,125 +9,146 @@ execute:
+It looks at all types of hospital visits, like surgeries, check-ups, and emergencies, and breaks it down by things like age, sex, and treatment.
 
 ```{r multiple}
 value_boxes <- list(
   bslib::value_box(
-    title = "Number of settings",
-    value = "95",
-    showcase = bsicons::bs_icon("gear"),
+    title = "NHS trusts using the model",
+    value = "27+",
+    showcase = bsicons::bs_icon("hospital"),
     theme = "primary"
   ),
   bslib::value_box(
-    title = "Years into the future",
-    value = "20",
+    title = "Looking ahead",
+    value = "2041/42",
     showcase = bsicons::bs_icon("graph-up"),
     theme = "primary"
   ),
   bslib::value_box(
-    title = "NHS trusts using it",
-    value = "20",
-    showcase = bsicons::bs_icon("hospital"),
+    title = "Customisation options",
+    value = "90+ ",
+    showcase = bsicons::bs_icon("gear"),
     theme = "primary"
   ),
   bslib::value_box(
-    title = "Rows of data",
-    value = "30 million",
+    title = "Rows of real hospital data",
+    value = "140 million",
     showcase = bsicons::bs_icon("journal-medical"),
-    theme = "primary",
+    theme = "primary"
   )
 )
 
+
 bslib::layout_column_wrap(
   width = "150px",
-  !!!value_boxes
+  !!!value_boxes[1:2]
 )
+
 ```
 
-## How Does the NHP Model Work?
+## What factors does it consider?
 
-### How it runs
+The tool considers many things that can change how much people need hospitals in the future:
 
-1. **Uses real data**: Based on 2019/20 hospital activity (HES data)  
-1. **Growth**: More people, living longer means more activity
-1. **User changes**: Enter expected changes for hospital activity, given as ranges
-1. **Simulations**: Runs hundreds of scenarios to explore possible futures 
+### Population changes
 
-### Parameters
+As the population grows, ages, and experiences changes in health, healthcare demand will change. We use ONS population projections and health trends to model how these factors will evolve over time.
 
-- Includes demographic and non-demographic growth.
-- Some parameters are nationally set and fixed.  
+### Non-demographic growth
 
-- **Demographic growth**: We use ONS population projections to see how the population will change. Taking into account that our _health status_ will also change.
+This refers to changes in healthcare demand not related to population size, age, or gender, such as:
 
-- **Non-demographic growth**: There are many other factors which might lead to hospital activity growing in future. We capture these and set this nationally.
+- Advances in medical technology
+- Changes in clinical standards
+- Rising patient expectations
 
-- **Mitigation**: The activity does not happen. E.g. fewer people smoking leads to fewer lung cancer treatment
+Our experts have analysed previous trends to predict how this growth will affect the future.
 
-- **Efficiencies**: As procedures become less invasive we might expect the length of time a patient stays in hospital to decrease.
+### Prevention 
```

Comment:
I'd maybe use a different word as not all mitigators are prevention. I think the word "mitigation" is relatively non-technical, or "reduction"?